cpp/src/io/parquet/parquet_gpu.hpp

```diff
@@ -228,7 +228,7 @@ struct PageInfo {
  * @brief Struct describing a particular chunk of column data
  */
 struct ColumnChunkDesc {
-  ColumnChunkDesc() = default;
+  constexpr ColumnChunkDesc() noexcept {};
```

Comment:
does `default` also not meet `14.5.3.1. Device Memory Space Specifiers` specifications?
is `default` not considered as trivial constructor if all non-static members have trivial constructors?

I see usage of `default` in this same PR.

Reply:
I need the constexpr constructor for the `byte_stream_s` type in page_hdr.cu. I agree that on initial read the `= default` constructor should be sufficient, but the compiler disagrees.